<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TBSLT00680" failures="5" tests="11" name="Test2" time="0.461" errors="0" timestamp="2024-07-23T14:13:24 IST">
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_HomePage" time="0.002"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Login" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [400] but found [300]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [300]
at ExtentReport.Facebook_Login.faceBook_Login(Facebook_Login.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- faceBook_Login -->
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Notification" time="0.003"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Profile" time="0.004"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Login" time="0.029">
    <failure type="java.lang.AssertionError" message="expected [400] but found [300]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [300]
at ExtentReport.Facebook_Login.faceBook_Login(Facebook_Login.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- faceBook_Login -->
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Registration" time="0.011"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_TimeLine" time="0.084"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_posts" time="0.005"/>
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Login" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [400] but found [300]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [300]
at ExtentReport.Facebook_Login.faceBook_Login(Facebook_Login.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- faceBook_Login -->
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Login" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [400] but found [300]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [300]
at ExtentReport.Facebook_Login.faceBook_Login(Facebook_Login.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- faceBook_Login -->
  <testcase classname="ExtentReport.Facebook_Login" name="faceBook_Login" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [400] but found [300]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [300]
at ExtentReport.Facebook_Login.faceBook_Login(Facebook_Login.java:27)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- faceBook_Login -->
</testsuite> <!-- Test2 -->
